ubuntu/python/README.md

```diff
@@ -0,0 +1,15 @@
+# Python Container
+
+This image adds a working Python environment to `databricksruntime/minimal` using virtualenv. For a Conda-based Python environment, an alternative image can be found [here](https://github.com/databricks/containers/tree/master/ubuntu/python-conda).
```

Comment:
@husseinnagr-db should we mention the conda alternative here or not, since we want users to switch to virtualenv?

Reply:
I think it's fine to mention it, but let's specify that Databricks recommends virtualenv? We should also mention that certain features like notebook-scoped libraries (%pip) will not work with the conda recipe in newer runtimes

Reply:
Additionally, could you update the first line of the README here - https://github.com/databricks/containers/tree/master/ubuntu/python-conda to say the same (that databricks recommends the default virtualenv image, and certain features like %pip won't work)